test/test_video_reader.py

```diff
@@ -1244,16 +1245,39 @@ def test_invalid_file(self):
         with self.assertRaises(RuntimeError):
             io.read_video('foo.mp4')
 
-    def test_audio_present(self):
-        """Test if audio frames are returned with video_reader backend."""
-        set_video_backend('video_reader')
+    def test_audio_present_pts(self):
+        """Test if audio frames are returned with pts unit."""
+        backends = ['video_reader', 'pyav']
+        start_offsets = [0, 1000]
+        end_offsets = [3000, None]
+        for test_video, _ in test_videos.items():
+            full_path = os.path.join(VIDEO_DIR, test_video)
+            container = av.open(full_path)
+            if container.streams.audio:
+                for backend, start_offset, end_offset in itertools.product(
+                        backends, start_offsets, end_offsets):
+                    set_video_backend(backend)
+                    _, audio, _ = io.read_video(
+                        full_path, start_offset, end_offset, pts_unit='pts')
+                    self.assertGreaterEqual(audio.shape[0], 1)
```

Comment:
This test is to make sure that we don't return an empty audio frame(as was the case previously). This makes sure that case would be checked.
We could compare the shape with pyav output, but I feel that's a test for the accuracy of results rather than just verifying if frames are returned.
Shape of audio frames would be different for different videos, so we could add ground truth values and compare against them, but I feel that's an overkill.
Feel free to share any suggestions you have.

Reply:
@datumbox I think we're ok here because this test only checks if the audio is present, so basically we just wanna check if _some_ frames are returned. There is a test for accuracy checking (provided that AFRAMES exist) [here](https://github.com/pytorch/vision/blob/4091d28e30c19b179bc9262102dc9c2a66097fe4/test/test_video_reader.py#L370)

Reply:
Sounds good to me.